.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Can we delete this file? It might cause problem with the setup.

Reply:
This file is the config file that *Gitpod* uses when you open the project in Gitpod and as as far as I have tested, doesn't meddle with the source files. Could you please elaborate on what files conflict with this one and which setup is being affected by this file? We could resolve the issue better that way.

The content of this file basically tells Gitpod to build the bundle (which is not pushed to Github, i.e. ignored) and also start the `dev` server so that we can preview the docs website. If you open the project on your machine locally by checking out the PR, this file is ignored and the project should open as it does normally.

Reply:
I am not familiar with the gitpod so thought it might conflict. Anyways nice addition to the playground. Let's wait for the authorized users to react to this.